src/ol/interaction/pinchzoom.js

```diff
@@ -29,6 +29,13 @@ ol.interaction.PinchZoom = function(opt_options) {
 
   /**
    * @private
+   * @type {boolean}
+   */
+  this.constrainResolution_ = options.constrainResolution ?
+options.constrainResolution : false;
```

Comment:
The second one is not only simpler and easier to read .. it also solves the indention problem ;)
Thanks for the notes!